src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -443,12 +448,23 @@ private static void validateGranularityRanges(List<ExtGranularityRange> ranges)
         }
     }
 
+    /**
+     * Validates {@link ExtGranularityRange}s max.
+     */
+    private static void validateGranularityRangeMax(BigDecimal rangeMax)
+            throws ValidationException {
+        if (rangeMax == null) {
```

Comment:
i'd recommend to place this verification inlined before `if (rangeMax.compareTo(nextRangeMax) > 0)`.